js/src/components/account-select-control/index.js

```diff
@@ -1,39 +0,0 @@
-/**
- * External dependencies
- */
-import { SelectControl } from '@wordpress/components';
-import { __ } from '@wordpress/i18n';
-
-/**
- * Internal dependencies
- */
-import AppSpinner from '.~/components/app-spinner';
-import toAccountText from '.~/utils/toAccountText';
-import './index.scss';
-
-const AccountSelectControl = ( props ) => {
```

Comment:
Checked and confirmed that there is no more `AccountSelectControl` in the code base.